ParallaxGenLib/src/ParallaxGenDirectory.cpp

```diff
@@ -155,13 +156,19 @@ auto ParallaxGenDirectory::mapFiles(const unordered_set<wstring> &NIFBlocklist,
       WinningSlot = NIFUtil::getSlotFromTexType(WinningType);
     }
 
+    bool IgnoreTexture = false;
+    if (isFileInBSA(Texture, BSAExcludes) && (WinningSlot == NIFUtil::TextureSlots::PARALLAX)) {
+      spdlog::trace(L"Ignoring height map {} from Vanilla BSA", Texture.wstring());
+      IgnoreTexture = true;
```

Comment:
I would just `continue` here and forget the bool. That way it won't log the message below and make it seem like it did actually map it.